spec.html

```diff
@@ -40856,9 +40857,9 @@ <h1>Module Namespace Objects</h1>
     <p>A Module Namespace Object is a module namespace exotic object that provides runtime property-based access to a module's exported bindings. There is no constructor function for Module Namespace Objects. Instead, such an object is created for each module that is imported by an |ImportDeclaration| that includes a |NameSpaceImport|.</p>
     <p>In addition to the properties specified in <emu-xref href="#sec-module-namespace-exotic-objects"></emu-xref> each Module Namespace Object has the following own property:</p>
 
-    <emu-clause id="sec-@@tostringtag">
-      <h1>@@toStringTag</h1>
-      <p>The initial value of the @@toStringTag property is the String value *"Module"*.</p>
+    <emu-clause oldids="sec-@@tostringtag" id="sec-%symbol.tostringtag%">
+      <h1>%Symbol.toStringTag%</h1>
```

Comment:
This should probably be wrapped in brackets:

```suggestion
      <h1>[ %Symbol.toStringTag% ]</h1>
```

Reply:
That's not the current convention, or the previous text; the property key is a Symbol, which is what `%Symbol.toStringTag%` is.